lib/utils/functionInfoUtils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+module.exports = {
+  aws: {
```

Comment:
is that a pattern we want to start using for provider specific utils? imo we should start keeping them in a separate directory to make it easy later on to move into its own package as planned. cc / @brianneisler